pkg/arm/v11/resources.go

```diff
@@ -265,6 +306,13 @@ func (g *simpleGenerator) ilbAPIServer() *network.LoadBalancer {
 		Location: to.StringPtr(g.cs.Location),
 	}
 
+	if g.cs.Properties.PrivateAPIServer {
```

Comment:
we only call ilbAPIServer() if PrivateAPIServer is set, fwiw, so these might go more clearly rolled in above.

Reply:
will do